tensorflow/contrib/BUILD

```diff
@@ -104,7 +105,9 @@ py_library(
         "//tensorflow/contrib/training:training_py",
         "//tensorflow/contrib/util:util_py",
         "//tensorflow/python:util",
-    ] + if_mpi(["//tensorflow/contrib/mpi_collectives:mpi_collectives_py"]),
+    ] + if_mpi(["//tensorflow/contrib/mpi_collectives:mpi_ops_py"])
```

Comment:
@samikama why changing this line?

Reply:
@aaroey isn't this coming from 98ffa7b2bb ? I thought you changed it.

Reply:
@samikama I thought it came from the first commit of trt?
https://github.com/samikama/tensorflow/blame/tensorrt/tensorflow/contrib/BUILD#L108

Reply:
Since we branched from around end of November, there was plenty of merge mistakes when we were creating the PR and had to resolve them manually. For this one, I must have taken the wrong one. We don't need any changes there.

Reply:
No problem, I'll fix that along with my fixes to the build deps.